src/components/__snapshots__/Card.spec.tsx.snap

```diff
@@ -131,14 +131,7 @@ exports[`TaskStepCard can optionally provide task 1`] = `
           </span>
         </div>
       </div>
-      <div>
-        <div
-          className="points"
-        >
-          1.0
-           Points
-        </div>
-      </div>
+      <div />
```

Comment:
Weird, now that I'm looking at this again, I dunno where the empty div is coming from 🤔